### PR TITLE
DPDK: Add epel install for RHEL vhds to install dpdk

### PR DESCRIPTION
**Pull request by mcgov**
RHEL images aren't guaranteed to have a DPDK package in their base repositories, add EPEL to install it from pkg manager if possible.

### CODE REVIEW

**Comment by LiliDeng** [comment]
thanks for the fix, please try to run some testing to verify it. 
https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/d/ doesn't contain the dpdk.
After enabling extra repo by steps in below link, I can install dpdk package dpdk.x86_64 0:18.11.8-1.el7_8.
The image I tested is redhat rhel 7.6 7.6.2019052206
https://gist.github.com/RulerOf/2bc34db224ccf8606e41b16c315d0bb6

File: lisa/operating_system.py [hunk 1]
```diff
@@ -1359,6 +1359,27 @@ def capture_system_information(self, saved_path: Path) -> None:
...
+            self._node.execute(
+                "subscription-manager repos "
+                f"--enable rhel-{version}-server-optional-rpms "
+                f"--enable rhel-{version}-server-extras-rpms",
+                sudo=True,
+            )
+        else:
+            self.install_packages("yum-utils")
+            self._node.execute(
+                "yum-config-manager --enable extras",
```
Comment (LiliDeng) [inline lisa/operating_system.py]: we have tool `YumConfigManager`

File: microsoft/testsuites/dpdk/dpdktestpmd.py [hunk 2]
```diff
@@ -791,6 +791,10 @@ def _install_fedora_dependencies(self) -> None:
         # update to latest kernel before instaling dependencies
         rhel.install_packages("kernel")
         node.reboot()
+        if isinstance(node.os, Redhat):
+            # rhel is not guaranteed to have a DPDK package in the default
+            # repos, add EPEL to get a DPDK version in the pkg manager.
+            node.os.install_epel()
```
Comment (LiliDeng) [inline microsoft/testsuites/dpdk/dpdktestpmd.py:808]: is it 'node.os.enable_extras()'?

Comment (mcgov) [reply microsoft/testsuites/dpdk/dpdktestpmd.py:808]: ah sorry I should have moved this to a draft, I was just editing and testing. I'll promote out of draft status when it's ready for review again.

**Comment by LiliDeng** [comment]
@mcgov could you please update this PR?

**Comment by LiliDeng** [comment]
feel free to create new one or reopen it when it is ready, thanks

**Pull request closed without merging**